public/api/conf/1.0/application.raml

```diff
@@ -0,0 +1,92 @@
+#%RAML 1.0
```

Comment:
have you double checked that this RAML will actually appear on developer hub ok? I wrote some documentation here for our API: https://github.com/hmrc/help-to-save-api#testing-the-raml-documentation-locally to test the RAML locally

Reply:
Yes - thank you - it now validates.